pkg/config/config.go

```diff
@@ -63,7 +64,7 @@ func (g YamlConfig) Parse() (*Config, error) {
 		sort.Slice(sections, func(i, j int) bool {
 			sectionI, sectionJ := sections[i].Type(), sections[j].Type()
 
-			if strings.Compare(sectionI, sectionJ) == 0 {
+			if strings.Compare(sectionI, sectionJ) == 0 && !g.Cfg.NoLexOrder {
```

Comment:
If enable, do not run sort anymore, check this in the sort function introduces more steps.

Reply:
Hi, will inverting the check as in https://github.com/daixiang0/gci/pull/210/commits/faf1c89519e591e89f177a2bde44b9692cea1b89 be ok? It is less steps as well after || short circuits